extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -6,7 +6,7 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 	'wikia.document',
 	'wikia.log',
 	'wikia.window'
-], function (googleSlots, helper, doc, log, window) {
+], function (googleSlots, recovery, doc, log, window) {
```

Comment:
This is not our change, but... 
Thanks! :D

we have already `sourcePointRecovery.js`